subsys/bluetooth/controller/Kconfig

```diff
@@ -345,6 +345,15 @@ config BT_CTLR_TIFS_HW
 	help
 	  Enable use of hardware accelerated tIFS Trx switching.
 
+config BT_CTLR_PA_LNA_GPIOTE_CHAN
```

Comment:
I think this belongs in the  "BLE Controller hardware configuration" section

Reply:
Allocation of GPIOTE should be similar to PPI allocation, hence instead of hardcoding the index in implementation, I used a Kconfig, hence it is hidden inside the advanced features. Will like to keep it hidden.

Reply:
That's fine, can't you make it hidden but in the hardware configuration section?